tasks/burrows-wheeler-transform/src/main.rs

```diff
@@ -0,0 +1,115 @@
+use core::cmp::Ordering;
+
+const STX: char = '\u{0002}';
+const ETX: char = '\u{0003}';
+
+pub fn special_cmp(lhs: &str, rhs: &str) -> Ordering {
```

Comment:
Can you add some docs to the function to explain its purpose?

Reply:
If you mean the cmp function, I have added some comments with the latest commit. Basically the special start and end chars (STX and ETX) should be sorted _after_ the alphanumeric characters, and not before.
If general the BWT transform should be described, let me know I can add that as well, with some links.

Reply:
Yep, I was just looking for some explanation on what makes it "special".